workshops/util.py

```diff
@@ -28,3 +34,75 @@ def earth_distance(pos1, pos2):
 
     # Multiply by 6373 to get distance in km.
     return arc * 6373
+
+
+def upload_person_task_csv(uploaded_file):
+    """
+    Read data from CSV and turn it into JSON-serializable list of dictionaries.
+    "Serializability" is required because we put this data into session.  See
+    https://docs.djangoproject.com/en/1.7/topics/http/sessions/ for details.
+    """
+    persons_tasks = []
+    reader = csv.DictReader(uploaded_file)
+    for row in reader:
+        person_fields = dict((col, row[col].strip())
+                             for col in PERSON_UPLOAD_FIELDS)
```

Comment:
Malformed files (try uploading, for example, a python script) will throw KeyError here: no key 'personal'.

I think the generator may look something like this:

```python
(col, row.get(col, "").strip() for col in PERSON_UPLOAD_FIELDS)
```

But I don't know if this makes any sense… A file without required columns?

Additionally, we need test for this case.